app/lucy/src/app/components/base-form/base-form.component.html

```diff
@@ -31,39 +87,95 @@
     </div>
   </div>
   <!-- -------------------- -->
-  <div class="container">
-    <br />
-    <div *ngFor="let section of config.sections; let i = index" class="container">
+  <br />
+  <div class="container form-body">
+    <div
+      class="container-fluid"
+      *ngFor="let section of config.sections; let i = index"
+    >
+      <br />
       <div class="row">
         <div class="col-12 section-title">
-          {{section["title"]}}
+          {{ section["title"].default }}
           <hr class="section-divider" />
         </div>
       </div>
-      <div *ngFor="let subSection of section.subSections; let j = index" class="row">
+      <div
+        *ngFor="let subSection of section.subSections; let j = index"
+        class="row"
+      >
+      
         <div class="col-12 sub-section-title">
-          {{subSection["title"]}}
+            <br/>
+          {{ subSection["title"].default }}
         </div>
-        <div class="col-12" [ngClass]="subSection.boxed ? 'boxed' : ''">
-          <div *ngFor="let field of subSection.fields; let j = index">
-
-            <app-field *ngIf="field.isInputField" [mode]="mode" [header]="field.header" [value]=""
-            (valueChanged)='fieldChanged(field, $event)'>
-            </app-field>
-            
-            <app-dropdown *ngIf="field.isDropdown" [mode]="mode" [items]="field.dropdown" [fieldHeader]="field.header"
-            [selectedItem]="organization" (selectionChanged)='fieldChanged(field, $event)'>
-            </app-dropdown>
-
-            <app-checkbox
-                *ngIf="field.isCheckbox"
+        <div class="container" [ngClass]="subSection.boxed ? 'boxed' : ''">
```

Comment:
You can bind properties  with [ngClass], try to use composite property. Like isBoxed: boolean